wegent-cli/wegent/commands/upgrade.py

```
@@ -0,0 +1,95 @@
+            click.echo(click.style("Error upgrading via git:", fg="red"), err=True)
+            click.echo(e.stderr, err=True)
+            sys.exit(1)
+
+    else:
+        # Assume Pip / PyPI install
+        click.echo("Detected Pip installation.")
+        package_name = "wegent-cli"  # Or whatever the package name on PyPI is/will be
+
+        try:
+            click.echo(f"Running pip install --upgrade {package_name}...")
+            subprocess.check_call(
```

Ruff (0.14.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)